src/components/MyOrder/MyOrderItem.tsx

```diff
@@ -1,12 +1,36 @@
-import { Flex, Box, Text, Stack, Badge } from "@chakra-ui/react";
+import { Flex, Box, Text, Stack, Badge, Button } from "@chakra-ui/react";
 import Image from "next/image";
 import formatCurrency from "@/utils/formatCurrency";
+import { downloadToPDF } from "@/utils/printConstruction";
```

Comment:
How about try lazy loading?

Reply:
Do you mean we can try lazy loading for items in the order list and cart list? I agree with that. Should we use a new card to add this new feature?

Reply:
We don't need a new card for this. Please look at our existing coding, there are lazy loading imports somewhere.